code/game/mecha/mech_fabricator.dm

```diff
@@ -135,7 +150,35 @@
 			update_static_data(user)
 		return
 	return ..()
-
+/**
+ * All the negative wire effect
+ * Zap doesnt actually zap, it breaks one limb (Because pain is to be had)
+*/
+/obj/machinery/mecha_part_fabricator/proc/wire_zap(mob/user)
+	if(stat & (BROKEN|NOPOWER))
+		return FALSE
+	var/datum/effect_system/spark_spread/s = new /datum/effect_system/spark_spread
+	s.set_up(5, 1, src)
+	s.start()
+	var/mob/living/carbon/C = user
+	var/datum/wound/blunt/severe/break_it = new
+	///Picks limb to break. People with less limbs have a chance of it grapping at air
+	var/picked_bodypart = pick(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_R_LEG, BODY_ZONE_L_LEG)
+	var/obj/item/bodypart/bone = C.get_bodypart(picked_bodypart)
+	if(bone){
+		to_chat(C,span_userdanger("The manipulator arms grapple after your [bone.name], attempting to break its bone!"))
+		break_it.apply_wound(bone)
+		bone.receive_damage(brute=50, updating_health=TRUE)
+	}
+	else{
+		to_chat(C,span_userdanger("The manipulator arms attempt to grab one of your limbs, but grapple air instead!"))
```

Comment:
delete the wound if you don't apply it

Reply:
Ok I won't resolve this one as I have no clue how Byond memory management works. I just used qdel() and hope it's correct

Reply:
Nvm got it